docs/source/common/lightning_cli.rst

```diff
@@ -354,23 +368,24 @@ before and after the execution of fit. The code would be something like:
         def add_arguments_to_parser(self, parser):
             parser.add_argument('--notification_email', default='will@email.com')
 
-        def before_fit(self):
+        def before_run(self):
```

Comment:
All the `*_run` in the documentation should be changed back to `*_fit`. And it should be explained that for every function there are the respective before and after methods.